src/lib/forms/Switch.svelte

```diff
@@ -81,14 +102,14 @@
     background-color: rgb(var(--m3-scheme-outline));
     cursor: pointer;
     -webkit-tap-highlight-color: transparent;
-    transition: all 300ms cubic-bezier(0.271, -0.011, 0, 1.449);
-
     left: 0.5rem;
     top: 50%;
     transform: translate(0, -50%);
     display: flex;
     align-items: center;
     justify-content: center;
+    transition-property: transform;
+    transition: 300ms cubic-bezier(0.271, -0.011, 0, 1.449);
```

Comment:
This seems questionable, won't this replace the previously declared transition name with the default (all) due to how CSS works?

Reply:
yeah but it looks fine. was anything else being transitioned?